plugins/services/src/js/components/__tests__/ConfigurationMapTable-test.js

```diff
@@ -132,7 +132,7 @@ describe("ConfigurationMapTable", function() {
     ).map(JestUtil.mapTextContent);
 
     expect(headerText).toEqual(["A", "B"]);
-    expect(cellText).toEqual(["1", "Not Configured"]);
+    expect(cellText).toEqual(["1", "\u2014"]);
```

Comment:
why not importing `#SRC/js/constants/EmptyStates` and using here?

Reply:
Yes I should do that, good catch.